client/js/app.js

```diff
@@ -17,6 +17,7 @@ import asynContent from './components/async-content';
 import contentSlider from './components/content-slider';
 import gaScrollDepth from '../libs/ga-scroll-depth';
 import joinCohort from './components/join-cohort';
+import './tracking';
```

Comment:
Could we `init` this for consistency (and it's always a little unclear that if you include something, it's run)